static/bidder-params/coinzilla.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Coinzilla Adapter Params",
+  "description": "A schema which validates params accepted by the Coinzilla adapter",
+  "type": "object",
+  "properties": {
+    "placementId": {
+      "type": "string",
+      "description": "Placement Id or Website Id"
```

Comment:
Updated with "minLength": 1 and test added

Reply:
@arkan303 Thanks for addressing all the comments. However, for the empty param test, I meant adding it in `invalidParams` in params_test.go. The test you currently added doesn't test the `minLength` setting on that parameter because the parameter validation is done before `MakeRequests` for your bidder is called and therefore JSON tests are not the right place to test that behavior unless you have specific parameter validation logic in the `MakeRequests` function. So can you please remove the empty_parameter.json test and it as part of `invalidParams` in params_test.go?

Thanks! I hope this makes sense.

Reply:
Update.Thanks!